CMP1903_A1_2324/Game.cs

```diff
@@ -8,14 +10,48 @@ namespace CMP1903_A1_2324
 {
     internal class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
+       public List<int> Roller()
+        {
+            int sum = 0;
+            List<int> returnList = new List<int>(); // a list created which holds the values that will be returned from this method
+            List<string> possibleAnswers = new List<string>() { "yes","no" }; // holds values of the two expected answers from the user when asked if they wish to continue
+            string answer = "yes";
+            while (answer != "no") // loops until user enters "no" after being asked if they wish to roll again
```

Comment:
You could seperate this large method into two smaller ones ;
Roller() and KeepRolling()?